<!DOCTYPE html>
<html lang="en">
	<head>
		<title>Juicy Brain</title>
		<meta charset="utf-8" />
		<script src="https://kit.fontawesome.com/4c3feccb25.js" crossorigin="anonymous"></script>
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>


					<nav id="nav">
					<ul>
						<li><a href="index.html">Home</a></li>
					</ul>
					</nav>


	<body class="homepage is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<header id="header">
					<div class="logo container">
						<div>
							<p> Curiosity and Passion </p>
						</div>
					</div>
				</header>

			<!-- Nav -->


			<!-- Banner -->
	<!--			<section id="banner">
					<div class="content">
						<h2>Greetings</h2>
					</div>
				</section>       -->    

			<!-- Main -->
				<section id="main">
					<div class="container">
						<div class="row gtr-100">
							<div class="col-12">

								<!-- Highlight -->
									<!--  <section class="box highlight">
										<header>
											<h2>Whole genome methylation profiling of rare neuronal subtype by Bisulphite-Sequencing and Enzymatic Methyl-Sequencing </h2>
											<p>  </p> 
										</header>
									-->

									<div class="col-3 col-6-medium col-12-small">

													<!-- Feature -->
													<section class="box feature">
													<p style="font-size: 26px;color: #000000; margin-bottom: 32px">Whole genome methylation profiling of rare neuronal subtype by Bisulphite-Sequencing and Enzymatic Methyl-Sequencing </p1>
													<p></p>
													<a href="images/D2.WGBS.EMseq_1.jpg" class="image featured"><img src="images/D2.WGBS.EMseq_1.jpg" alt="" /></a>
<!-- 												<h3><a href="#">Snot shooter</a></h3> -->
													</section>
												</div>																				
										<p>
										We have developed two protocols to comprehensively profile the rare neuronal population from individual mouse brains. One is based on bisulfite treatment for whole-genome bisulfite sequencing (WGBS), and the other utilizes Enzymatic Methyl-Sequencing (EMseq from New England Biolabs). We profiled the DNA methylation of Drd2-expressing neurons in the nucleus accumbens, which is an important brain region for reward processing and addiction.
										</p> 																			
											<div class="col-3 col-6-medium col-12-small">
													<!-- Feature -->
														<section class="box feature">
														<p style="font-size: 26px;color: #000000; margin-bottom: 32px">Distinct bias between WGBS and EMseq</p1>
														<p></p>
															<a href="images/D2.WGBS.EMseq_2.jpg" class="image featured"><img src="images/D2.WGBS.EMseq_2.jpg" alt="" /></a>
<!-- 															<h3><a href="#">Snot shooter</a></h3> -->
												
														</section>

												</div>
										
										
										<p>
										Our data show that WGBS and EMseq have distinct biases regarding unconverted cytosine and sequencing coverage, among other factors. These differences may originate from the distinct chemistry between these two methods. More CA dinucleotides were unconverted by bisulfite treatment, while CC and CG showed higher unconversion rates in EMseq data (a), which might be due to the lower accessibility of enzymes to GC-rich DNA strands. Likewise, WGBS is more likely to yield lower coverage in GC-rich regions (b,c). Although EMseq shows less systemic bias compared to WGBS, the biases of EMseq are not consistent across replicates. In other words, WGBS shows higher biases in several metrics, such as higher unconversion rates and lower coverage of high GC regions; however, EMseq shows greater variation between replicates.
										
										</p> 
										
										
											<div class="col-3 col-6-medium col-12-small">

													<!-- Feature -->
														<section class="box feature">
														<p style="font-size: 26px;color: #000000; margin-bottom: 32px">D2-MSN specific DNA methylation landscape</p1>
														<p></p>
															<a href="images/D2.WGBS.EMseq_3.png" class="image featured"><img src="images/D2.WGBS.EMseq_3.png" alt="" /></a>
<!-- 															<h3><a href="#">Snot shooter</a></h3> -->
												
														</section>

												</div>
										<p>
											We also analyzed cell type-specific differentially methylated regions (DMRs) by comparing D2-MSN to neural progenitor cells and Camk2a-expressing neurons from the mouse cortex. We pinpointed the transcription factors that might interact with the D2-MSN DMRs (a). We found that the cell identity genes, such as the dopamine receptor Drd2 and the opioid enkephalin preproprotein gene Penk, exhibit concurrent low CG and non-CG methylation (b). Taken together, this study provides insights into both DNA profiling and neuroepigenomics in addiction.
										</p> 
										
										
										<ul>
											<li> <a href="https://pubmed.ncbi.nlm.nih.gov/35205351/">Related publication 1</a>	</li>
										</ul>
										
										</div>
										<div class="col-9 col-6-medium col-12-small">
								
										<body>
										<button onclick="window.location.href='NAc.single.cell.html'"> NAc D1-/D2-MSN </button>
										</body>	
									<div>


</body>
</html>